pkg/operator/api/api.go

```diff
@@ -160,6 +160,8 @@ func (c *Cluster) makeDeployment() *extensions.Deployment {
 
 func (c *Cluster) apiContainer() v1.Container {
 
+	securityContext := k8sutil.BuildSecurityContext(false, 0, false, false)
```

Comment:
Nit: i would add comments to these args for better readability.
For example 
```securityContext := k8sutil.BuildSecurityContext(false /* not privileged */, 0, false /* run as root */, false /* RW filesystem */)```

Reply:
this is where C#'s "named arguments" are so useful (https://stackoverflow.com/a/20601083/467462).  i wish golang had something similar.

Reply:
I miss that from python too.

Reply:
@phsiao i dont mind this too much. That is why is said it was a nit :).
It just makes it a bit more clear to me what the arguments mean without having to go the function definition.

Reply:
Simple change for the better, happy to do it.  :)